ARCS/Api/Donate.cs

```diff
@@ -42,7 +67,9 @@ public DonationProgress(string targetPrefix)
                 {
                     Current = Max;
                 }
-                Percent = Math.Round((Current / Max) * 100,2);
+                double Percent = Math.Round((Current / Max) * 100,2);
```

Comment:
You should merge latest master in your branch. The founding has change to 0 decimal numbers